os/net/lwip/src/include/lwip/pbuf.h

```diff
@@ -227,6 +227,13 @@ void pbuf_free_ooseq(void);
 #define PBUF_CHECK_FREE_OOSEQ()
 #endif							/* LWIP_TCP && TCP_QUEUE_OOSEQ && NO_SYS && PBUF_POOL_FREE_OOSEQ */
 
+/* use in pbuf testcases */
```

Comment:
Is this necessary here? not in test file?

Reply:
yes, it is helpful here, from this comment anyone can get the idea why it is here.